wal/metrics.go

```diff
@@ -17,6 +17,13 @@ package wal
 import "github.com/prometheus/client_golang/prometheus"
 
 var (
+	syncGauge = prometheus.NewGauge(prometheus.GaugeOpts{
+		Namespace: "etcd",
+		Subsystem: "disk",
+		Name:      "wal_fsync_duration_current",
```

Comment:
@fabxc are we leaning towards possibly reopening this :)?  if so i'll name it anything you want :):):) would love to have this gauge !